packages/big-design/src/components/TableNext/hooks/useSelectable/helpers.ts

```diff
@@ -0,0 +1,84 @@
+import { TableSelectable } from '../../types';
```

Comment:
Please review this file

Reply:
i think if we change `selectedItems` data structure, this logic could be simplified. I'd like to hear what others have to think about that tho 🤔